Update sandbox teardown steps (#158)
GSA/notifications-api: docs/deploying.md

Rules for use:

1. Ensure that no other developer is using the environment, as there is nothing stopping changes from overwriting each other.
1. Clean up when you are done: 
    - `terraform destroy` from within the `terraform/sandbox` directory will take care of the provisioned services

>     - Delete the routes shown in `cf routes`
>     - Delete the apps shown in `cf apps`

Reviewer: [suggestion] I think we can simplify this:

```suggestion
    - Delete the apps and routes shown in `cf apps` by running `cf delete APP_NAME -r`
```

>     - Delete the service keys for any remaining space deployers, likely `cf dsk <space-deployer> service-account-key`
>     - Delete the space deployers still shown in `cf services`

Reviewer: [suggestion] I think this can be simplified by pointing people to `destroy_service_account.sh`

```suggestion
    - Delete the space deployers still shown in `cf services` by running `terraform/destroy_service_account.sh -s notify-sandbox -u <space-deployer>`
```
